Dependencies/d3dx.ini

```diff
@@ -343,8 +344,21 @@ analyse_options = log dump_rt_jps clear_rt
 [Device]
 
 ; overrides screen resolution.
-;width=1280
-;height=720
+; if upscaling is on the resizing functionality is disabled
+; however if upscaling is not zero then you HAVE to specify the resolution the game will be upscaled to
+; to achive the best visual result use the native resolution of your tv or monitor
+;width=2560
+;height=1440
+
+; (0) - disable upscaling; (1) - enable upscaling and allows the game to disable and enable fullscreen mode
```

Comment:
I'd move this section above the width and height parameters, because they interact.